keyboards/rotary_numpad/info.json

```
@@ -0,0 +1,64 @@
+{
+    "manufacturer": "rabijl",
+    "keyboard_name": "rotary_numpad",
+    "maintainer": "rabijl",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": ["F5", "F4", "B5", "B4"],
+        "rows": ["F6", "F7", "B1", "B3", "B2", "B6"]
+    },
+    "processor": "atmega32u4",
```

pro micro MCU daughter board used (as mentioned in `readme.md`)
```suggestion
    "development_board": "promicro",
    "diode_direction": "COL2ROW",
    "features": {
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
        "encoder": true
    },
    "matrix_pins": {
        "cols": ["F5", "F4", "B5", "B4"],
        "rows": ["F6", "F7", "B1", "B3", "B2", "B6"]
    },
```

---

Oh I was not aware that was possible with the info.json and couldn't find it in the documentation either.